src/features/document/document_feature.py

```diff
@@ -77,6 +85,10 @@ def update(
 )
 @create_response(PlainTextResponse)
 def remove(data_source_id: str, dotted_id: str, user: User = Depends(auth_w_jwt_or_pat)):
+    """Remove document
+
+    - **dotted_id**: uuid for document to delete
```

Comment:
```suggestion
    - **dotted_id**: uuid for document to delete
```
Dotted id should not be "uuid", but a path

Reply:
or {id}.attribute
Think that is the case here